admin/installer/setup_volumehub.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Set up Volume Hub if a token has been supplied.
+set -ex
+
+: ${volumehub_token:?}
+: ${node_number:?}
+
+if test -z "${volumehub_token}"; then
+    # No Volume Hub token supplied.
+    exit
+fi
+
+control_service=false
+agent_node=false
+agent_number="0"
+
+if test "${node_number}" -eq "0"; then
+    control_service=true
+else
+    agent_node=true
+    # This only works because we start numbering CloudFormation nodes at 0.
```

Comment:
Is that reliable, or is it likely to change? I don't know if there is a nice and reasonably straight-forward alternative.